impl/src/main/java/org/jboss/weld/event/ObserverNotifier.java

```diff
@@ -324,7 +330,7 @@ public RuntimeException apply(Type eventType) {
         }
         // We should always initialize and validate all notification options first
         final NotificationMode mode = initMode(options.get(WeldNotificationOptions.MODE));
-
+        final Long timeout = verifyNotificationTimeoutOption(options);
```

Comment:
Yes, wanted to ask about this.
I didn't like the `init` one because I needed to do not only option check, but also check for the executor presence.
If you don't mind, I will switch both methods to `verify` format.